### PR TITLE
#4 Issue Solved: Column size for name is too long when I configure on…

### PR DESCRIPTION
**Pull request by priyalpepcus**
…ly few columns to show managed using width in student grid page

### CODE REVIEW

File: src/components/DataGrid.jsx [hunk 1]
```diff
@@ -133,6 +137,46 @@ class StudentInfoGrid extends Component {
       }
     }
   }
+  componentDidUpdate() {
+    this.manageStudentTableWidth();
+  }
+  manageStudentTableWidth = () => {
```
Comment (mohitsaladiya) [inline src/components/DataGrid.jsx]: Please migrate this method to a util file.

File: src/components/DataGrid.jsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ import {
 class StudentInfoGrid extends Component {
```
Comment (mohitsaladiya) [inline src/components/DataGrid.jsx]: class name and file name should not be different. Please rename this file to 'StudentInfoGrid.jsx' and replace it on all the occurrences.

File: src/components/DataGrid.jsx (hunk 1 above)
Comment (priyalpepcus) [reply src/components/DataGrid.jsx]: done.

File: src/components/DataGrid.jsx (hunk 2 above)
Comment (priyalpepcus) [reply src/components/DataGrid.jsx]: Replaced.

**Pull request merged**